Gemfile

```diff
@@ -69,6 +69,9 @@ gem 'simple_form'
 # Passing controller data to JS
 gem 'gon'
 
+# JE Todo: Confirm if there are any potential conflicts between gon and active_model_serializers
+gem 'active_model_serializers'
```

Comment:
Don't think so. I'd like to transition away from gon eventually also. I'm not really a fan of how it acts like a big fat global variable.